src/Saritasa.NetForge.DomainServices/AdminOptionsBuilder.cs

```
@@ -100,6 +101,32 @@ public AdminOptionsBuilder AddGroups(IEnumerable<EntityGroup> groups)
         return this;
     }
 
+    /// <summary>
+    /// Configure the url to an external site.
+    /// </summary>
+    /// <param name="url">The url to be configured.</param>
+    public AdminOptionsBuilder SetSiteUrl(string url)
+    {
+        if (string.IsNullOrEmpty(url))
+        {
+            return this;
+        }
+
+        if (IsUrlValid(url))
```

Let's raise the exception in case the URL is invalid.